pubspec.yaml

```
@@ -10,16 +10,16 @@ environment:
   mockito: ^5.4.2
   build_runner: ^2.4.5
-  crypto: ^3.0.6
-  http: ^1.0.0
-  intl: ^0.19.0
-  collection: ^1.17.0
+  flutter_lints: '>=3.0.0 <4.0.0'
+  http_parser: '>=4.0.2 <5.0.0'
+  meta: '>=1.16.0 <2.0.0'
+  async: '>=2.11.0 <3.0.0'
+  crypto: '>=3.0.6 <4.0.0'
+  http: '>=0.13.0 <2.0.0' 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)